pinc/bootstrap.inc

```diff
@@ -43,13 +43,15 @@ if (!headers_sent()) {
 include_once($relPath.'gettext_setup.inc');
 include_once($relPath.'DPDatabase.inc');
 
-try {
-    DPDatabase::connect();
-} catch (Exception $e) {
-    // If we're in maintenance mode, don't die here - we'll more gracefully
-    // error out later
-    if (!$maintenance) {
-        throw $e;
+if (!defined('DISABLE_DATABASE_CONNECT_CHECK')) {
```

Comment:
This isn't actually just a check it's the sole connection to the DB.
```suggestion
if (!defined('SKIP_DB_CONNECT')) {
```

Reply:
Noted!